src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -15,6 +15,7 @@
 import java.util.Set;
 import java.util.StringTokenizer;
 import java.util.concurrent.atomic.AtomicReference;
+
```

Comment:
@DanVanAtta This is what I meant about "ping-ponging" in [#1876](https://github.com/triplea-game/triplea/pull/1876#discussion_r121290902).